src/python/grpcio/grpc/_adapter/_low.py

```diff
@@ -195,26 +195,30 @@ def start_batch(self, ops, tag):
         translated_op = cygrpc.operation_send_initial_metadata(
             cygrpc.Metadata(
                 cygrpc.Metadatum(key, value)
-                for key, value in op.initial_metadata))
+                for key, value in op.initial_metadata),
+            op.write_flags)
```

Comment:
It's a code smell that something called `write_flags` is being passed to all these different functions that have nothing to do with writing (and one function that does). Is the name `flags` available for the attribute being passed on the changed lines in this file?